sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -284,6 +284,10 @@ def from_dictionary(cls, options):
           flags.append('--%s=%s' % (k, i))
       elif isinstance(v, dict):
         flags.append('--%s=%s' % (k, json.dumps(v)))
+      elif v is None:
```

Comment:
Can you also write a small test for this `None` type args?

Reply:
Added a unit test case and confirmed it fails without and passes with my change in my local env.